.eslintrc.js

```diff
@@ -75,7 +75,8 @@ const dictionary = [
 
 // list of words that should be capitalized in a specific way
 const capitalized = [
-  'OAuth2'
+  'OAuth2',
+  'AiBuilderModel'
```

Comment:
Instead of adding the command name here, we should add the separate words to the dictionary: ai, builder and model